code/modules/mob_spawn/ghost_roles/spider_roles.dm

```diff
@@ -75,9 +75,12 @@
 	var/obj/structure/spider/eggcluster/egg
 	/// The types of spiders that the spawner can produce
 	var/list/potentialspawns = list(
-		/mob/living/basic/giant_spider,
-		/mob/living/basic/giant_spider/hunter,
-		/mob/living/basic/giant_spider/nurse,
+		/mob/living/basic/spiderling/nurse,
+		/mob/living/basic/spiderling/hunter,
+		/mob/living/basic/spiderling/ambush,
+		/mob/living/basic/spiderling/tangle,
+		/mob/living/basic/spiderling/guard,
+		/mob/living/basic/spiderling/scout,
```

Comment:
As we spoke about in discord, you need to refactor the selection menu so that it displays the grown spider by reading from `spiderling:grow_as` instead of showing a radial menu of 6 nearly identical spiderlings differentiated only by colour.

Reply:
![image](https://github.com/tgstation/tgstation/assets/84478872/1d612d5f-1d4f-4644-bd9e-8045da04138b)
I believe is a better design choice the spiderlings are both very easy to tell  a part show the info menu bar and show the name of what it is on the top screen

I think spawns should show you what you will spawn as an not what you will become is fully cosmetic anyways

I also prefer the look

Reply:
You aren't choosing a spiderling because you want to play as a spiderling, you are choosing it for what it grows into. That is the important information.
The look is worse because it is less informative than showing you the fully grown spiders, which are different in more than colour.

This UX is significantly worse, and not acceptable.

Reply:
I don't know what the UX is but fine